src/test/java/cloud/fogbow/ras/core/plugins/interoperability/azure/util/AzureStateMapperTest.java

```diff
@@ -0,0 +1,52 @@
+package cloud.fogbow.ras.core.plugins.interoperability.azure.util;
+
+import cloud.fogbow.ras.api.http.response.InstanceState;
+import cloud.fogbow.ras.core.models.ResourceType;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AzureStateMapperTest {
+
+    // test case: When calling the map method with compute type and creating state,
+    // it must verify if It returns the instance creating state.
+    @Test
+    public void testMapSuccessfullyWhenCreatingState() {
+        // set up
+        ResourceType resourceType = ResourceType.COMPUTE;
+
+        // exercise
+        InstanceState instanceState = AzureStateMapper.map(resourceType, AzureStateMapper.CREATING_STATE);
+
+        // verify
+        Assert.assertEquals(InstanceState.CREATING, instanceState);
+    }
+
+    // test case: When calling the map method with compute type and creating state,
```

Comment:
Test case is explained to the wrong case. It is set to **creating** state when it should be to the **succeeded** state